src/ripe_rainbow/domain/base/assertions.py

```diff
@@ -28,14 +28,17 @@ def has_text(self, selector, text):
 
         element = self.exists(selector)
 
-        if element and not element.text == text:
-            self.logger.debug("Element '%s' found but has text '%s' instead of '%s'" % (
-                selector,
-                element.text,
-                text
-            ))
-
-            return None
+        if element:
+            self.driver.scroll_to(element)

```

Comment:
I'm confused about this, the text attribute of the element may change with scrolling?